src/components/HeroSlider/HeroSlider.tsx

```diff
@@ -0,0 +1,68 @@
+import React from 'react';
+import Slider from 'react-slick';
+
+import 'slick-carousel/slick/slick.css';
+import 'slick-carousel/slick/slick-theme.css';
+import './HeroSlider.scss';
+import { picturesPath } from './picturesPath';
+
+interface Props {
+  className?: string;
+  style?: React.CSSProperties;
+  onClick?: () => void;
+}
+
+const CustomNextArrow: React.FC<Props> = ({ className, style, onClick }) => {
+  return (
+    <div
+      className={`${className} custom-arrow custom-arrow--next`}
```

Comment:
change all classNames to use Module CSS

Reply:
I can't use Module CSS, because I use the library 'react-slick'. Inside it use classes which use on page, but which I don`t write state obviously. Additionally,  'react-slick' use classes names, for example 'custom-arrow'. But this names I can`t use in Module CSS.